### DISCUSSION TITLE
Online Conversations (Discord,Slack,etc)

### DISCUSSION THREAD

**Discussion by ghost** [DavidRieman/WheelMUD#113] [Ideas]
Should we create a platform for better banter? This will allow easier back and forth conversations.

Discord or Slack would be a great addition ;)

**Comment by DavidRieman** [comment]
Agreed; looking into this.

**Reply by DavidRieman** [reply]
Created an open "WheelMUD" Discord server ([invite link here](https://discord.gg/7smhbsNKRZ)).

I've barely touched Discord a few times before, but been meaning to check it out more thoroughly anyways.  I think we want to start with only a few channels and see if it grows organically.
